PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -89,8 +89,36 @@ public static Question[] LoadQuestions(string filePath)
                 question.Answers[1] = answer2;
                 question.Answers[2] = answer3;
                 question.CorrectAnswerIndex = correctAnswerIndex;
+                RandomizeAnswerOrder(question);
+                questions[i] = question;
             }
             return questions;
         }
+
+        public static void RandomizeAnswerOrder(Question question)
+        {
+            Random random = new Random();
```

Comment:
There is another subtle issue that could arise here. Random is initialized with the current ticks property. Most of time your code will be fine, but there is a chance based on runtime, and OS, that you might end up initializing two `Random` instances with the same seed, and end up with multiple questions that get the same "random" order. For this application is doesn't really matter, just something to keep in mind. Often people will create a static Random instance to avoid this problem (just keep in mind that Random is not thread safe).